upgrade/helpers/tasks.py

```diff
@@ -1284,3 +1272,19 @@ def job_execution_time(task_name, start_time=None):
     else:
         start_time = datetime.now().replace(microsecond=0)
         return start_time
+
+
+def resume_failed_task():
+    """
+    This function is used to resume the canceled paused tasks.
+    """
+    output = run("hammer task list|awk '/Task canceled/ && /paused/ {print $1}'")
+    run(f"hammer task resume --task-ids {','.join(output.split())}")
+    logger.info(f"The CV publish task {','.join(output.split())} has resumed successfully, "
```

Comment:
If this is an universal function to resume the paused tasks, there is no point in logging the CV Publish here. You can do this logging after calling this function in a code.

Reply:
yes, you are right, I missed it.